### ISSUE TITLE
The macOS app store still shows the old version, the ios app store shows the new version.

### ISSUE THREAD

**Issue by x86-x86** [ursuscamp/nostore#22]
The macOS app store still shows the old version 1.1.0 , the ios app store shows the new version 1.2.0.

Can you also update the macOS app in the app store?

**Comment by btcforplebs** [comment]
I second this. MacOS App Store version needs updated. it does not function

**Comment by Sjors** [comment]
The macOS App Store doesn't show me a version number anywhere, but it indeed still appears to be the broken previous release.

**Comment by ursuscamp** [comment]
Yes, the app doesn't actually work on macOS unfortunately, since @fishcakeday updated it. This app is currently in a mostly unmaintained state as I am no longer a nostr user. If anyone would like to submit a PR to fix the app on MacOS, I will publish it. Otherwise, it may be a while.

**Comment by x86-x86** [comment]
The version that is uploaded in testflight, i.e. 1.2.0 works perfectly on macOS 15.

You just have to publish this version in the macOS app store.

**Comment by ursuscamp** [comment]
I did, Apple rejected it because it didn't work (UI non-responsive). I can verify that when I test it. However, my Mac is not yet updated to macOS 15. Perhaps if I resubmit to Apple with an updated target version of macOS 15 they will accept it.

**Comment by ursuscamp** [comment]
Check for it now

**Comment by btcforplebs** [comment]
Yo! There it is 🙌 thanks!

**Comment by Sjors** [comment]
Yup, I see the update now!

Sadly it's now marked as macOS 15.0 only, so doesn't work on my ancient iMac (but it does on my Macbook). Opened #23 for that.
